api/src/chat/controllers/block.controller.spec.ts

```diff
@@ -116,6 +128,12 @@ describe('BlockController', () => {
         PermissionService,
         LanguageService,
         PluginService,
+        LoggerService,
```

Comment:
```suggestion
```
Hi @MohamedAliBouhaouala, we can remove the LoggerService since it's automatically injected when we call buildTestingMocks function

![image](https://github.com/user-attachments/assets/c5065877-b04b-42b0-8296-fedd59a6313f)